### ISSUE TITLE
Guide Maven users how to configure concat for reference.conf instead of replace, in maven assembly

### ISSUE THREAD

**Issue by ktoso** [akka/akka#18813]
(Same applies to Gradle).

People doing fatjars, coming from java-land, end up sometimes getting config blow up since the default strategy of "2 names with the same name" is "replace" in those tools (Maven, Gradle), which does not make sense for reference.conf.

**Comment by ktoso** [comment]
Note to self, maybe we add a better error message for setting not found as well?

**Comment by ktoso** [comment]
I.e. improve exception to point to: http://doc.akka.io/docs/akka/2.4.0/general/configuration.html#When_using_JarJar__OneJar__Assembly_or_any_jar-bundler

**Comment by viktorklang** [comment]
even better—point to the reference docs for the version of akka it uses :)

**Comment by ktoso** [comment]
Yes, though it does need a fallback (to `current` I'd say) - since the version is stored in `reference.conf` which is the file we may be failing because right now (i.e. persistence's reference overwrote actors' reference) :+1:

**Comment by viktorklang** [comment]
@ktoso But you are wrong :) see ActorSystem.Version

**Comment by ktoso** [comment]
Oh ok, I grepped but only noticed `ConfigVersion`, great so we can link to the right one.
